CMSgov/eAPD
# Bbrooks/3835 update mongo script

Resolves # 3835

### Description
Adds mechanism for the creation of Mongo resources with New Relic infrastructure monitoring key populated as a part of the process. 

### Signifcant changes or possible side effects
Updates terraform and packer scripts as well as touches user-data scripts.

### Automated test cases written

### Steps to manually verify this change

1. Log into the eAPD Preview MongDB instance and check the newrelic-infra.yml file for the token
2. Make sure that communication to New Relic is successful
3.

### This pull request is ready to review when


### This pull request can be merged when
